app/controllers/application_controller.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
```

Comment:
I haven't seen this before. What's `devise_parameter_sanitizer` do that [strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters) don't